conf/genomes.config

```diff
@@ -27,6 +27,7 @@ params {
       knownIndelsIndex      = "${params.genomes_base}/{1000G_phase1,Mills_and_1000G_gold_standard}.indels.b37.vcf.idx"
       snpeffDb              = "GRCh37.75"
       vepCacheVersion       = "95"
+      species               = 'homo_sapiens'
```

Comment:
why single quotes?
Or why do we have double quotes for the rest?

Reply:
Ok, I adjusted them in both `igenomes.config` and `genomes.config` to keep things consistent :-)